pkg/Library/Graph/Nodes/LayerNode.js

```diff
@@ -6,7 +6,7 @@
 export const LayerNode = {
   Layer: {
     type: '$library/Graph/Atoms/Layer',
-    inputs: ['graph', 'graphJson', 'selectedProps', 'designable'],
+    inputs: ['graphId', 'graph', 'selectedProps', 'designable'],
```

Comment:
I hope you don't mind me renaming Layer's inputs
graph -> graphId
graphJson -> graph

(because it was confusing for me)